labgrid/remote/client.py

```diff
@@ -915,6 +915,29 @@ def flashscript(self):
 
         drv = self._get_driver_or_new(target, "FlashScriptDriver", name=name)
         drv.flash(script=self.args.script, args=self.args.script_args)
+    
+    def debugger(self):
+        place = self.get_acquired_place()
+        target = self._get_target(place)
+        name = self.args.name
+        from ..resource.lauterbach import (NetworkLauterbachDebugger, NetworkUSBLauterbachDebugger)
+        from ..resource.udev import (USBLauterbachDebugger)
+
+        drv = None
+        try:
+            drv = target.get_driver("DebuggerProtocol", name=name)
+        except NoDriverFoundError:
+            for resource in target.resources:
+                if isinstance(resource, (NetworkLauterbachDebugger, NetworkUSBLauterbachDebugger, USBLauterbachDebugger)):
+                    drv = self._get_driver_or_new(target, "LauterbachDriver", activate=False,
+                                                  name=name)
+                if drv:
+                    break
+
+        if not drv:
+            raise UserError("target has no compatible resource available")
+        target.activate(drv)
+        drv.start()
```

Comment:
```suggestion
    
    def debugger(self):
        place = self.get_acquired_place()
        target = self._get_target(place)
        name = self.args.name

        drv = self._get_driver_or_new(target, "LauterbachDriver", name=name)
        drv.start()
```
This should be sufficient.